Shopping-cart/Helper/ShoppingListServer.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ShoppingListServer.swift
+//  Shopping-cart
+//
+//  Created by cai dongyu on 2022/12/9.
+//
+
+import Foundation
+import Combine
+
+struct ShoppingListServer <T> where T: Decodable {
+  let url: String
+  func getDataFromRemote() -> AnyPublisher <Array<T>, Error> {
```

Comment:
Could we make the url be parameter of the function rather than property in the struct?